### DISCUSSION TITLE
FYI - Flash Attention 2 is supposedly faster, no idea if it beats xformers though

### DISCUSSION THREAD

**Discussion by ilikenwf** [vladmandic/sdnext#1732] [General]
https://twitter.com/_akhaliq/status/1680988185776607237

https://tridao.me/publications/flash2/flash2.pdf

https://github.com/Dao-AILab/flash-attention

**Comment by drax-xard** [comment]
According to the tweet it does, and by a pretty good margin too!

**Comment by ilikenwf** [comment]
It appears to be in xformers now? @vladmandic any thoughts?

I realize tensorrt is going to be faster but also a pain in the rear.
